tests/Pest/PreviewifyTest.php

```diff
@@ -20,14 +20,14 @@
     seo()->previewify('blog', 1);
     expect(seo()->previewify('blog', ['title' => 'abc', 'excerpt' => 'def']))
```

Comment:
If we change this line to this:
```suggestion
    expect(seo()->previewify('blog', ['title' => 'abc', 'previewify:excerpt' => 'def']))
```
Then I think the test will also ensure that `previewify:` is added _only_ when needed.